### ISSUE TITLE
enable editing remote database name in sync settings

### ISSUE THREAD

**Issue by toyfoo** [ajboni/drafft-docs#15]
for purposes of syncing flexibly, it would be useful to be able to change the remote database name. now, the only way to edit a remote database name is to change it in a json export and re-import the file.

**Comment by ajboni** [comment]
this is tricky, as we are using the project name to create/sync the remote database. Maybe loose this relationship.

**Comment by toyfoo** [comment]
I understand. But it's a limitation if you want to create a remote backup, relink, have another version,... and you already have a project with a similar name. because then you have to delete your local project to be able to sync with the remote one.

**Comment by ajboni** [comment]
> I understand. But it's a limitation if you want to create a remote backup, relink, have another version,... and you already have a project with a similar name. because then you have to delete your local project to be able to sync with the remote one.

Yes, I think this behavior was planned like this to avoid someone mistakenly changing and corrupting another similar named project. For example a typo may cause overwriting of another project.  But I agree this should be more flexible / handled differently.

**Comment by toyfoo** [comment]
maybe a check, that if data already exists at the remote server, the application asks if the local database can be emptied first. and at that time the name is locked. maybe that is a middle way between locking the user completely out of different names for remote db and local db, but ensuring that once a sync has been established it is not messed up.

**Comment by ajboni** [comment]
>  but ensuring that once a sync has been established it is not messed up.

I like it, thanks for the idea.
